extensions/deploy_replace_undeploy/standard/recommendations/deploy-replace-undeploy/replace/REC_body-argo.adoc

```diff
@@ -0,0 +1,10 @@
+[[rec_deploy-replace-undeploy_replace_body-argo]]
+[recommendation]
+====
+[%metadata]
+label:: /rec/deploy-replace-undeploy/replace/body-argo
+
+part:: If a process can be described for the intended use as a <<rc_argo,Argo graph>>, implementations should consider supporting the <<rc_argo,Argo>> encoding for describing the replacement process.
+
+part:: The media type `application/argo` shall be used to indicate that request body contains a processes description encoded as <<rc_ogcapppkg,Argo>>.
```

Comment:
Is `application/argo` an official media-type? If not, the generic https://www.iana.org/assignments/media-types/application/vnd.oai.workflows+yaml with a `contentSchema` with the Argo Workflow schema URL might be more appropriate.

An alternative would be to push Argo maintainers to publish a media-type like CWL did:

https://www.iana.org/assignments/media-types/application/cwl

https://www.iana.org/assignments/media-types/application/cwl+json